index.js

```diff
@@ -0,0 +1,50 @@
+import Bookstore from './modules/books.js';
+import formatAMPM from './modules/time.js';
+import SetupTabs from './modules/switch.js';
+
+const author = document.querySelector('#author');
+const title = document.querySelector('#title');
+const addBookForm = document.querySelector('#add-book');
+const list = document.querySelector('.list');
+const time = document.querySelector('.time');
+// /*eslint-disable*/
+// let luxon;
+// const { DateTime } = luxon;
+// /*eslint-disable*/
```

Comment:
- [ ] Please do not leave unused code as a comment in your codebase. unused code as comments makes your code not clean. Kindly remove all unused code this will make your code mush noise, It is always a good practice to keep your code clean from such comments.